aspnetcore/security/authentication/sociallogins.md

```diff
@@ -143,28 +124,31 @@ Once you enter your Facebook credentials, then you will be redirected back to th
 
 You are now logged in using your Facebook credentials.
```

Comment:
Replace these two Facebook-specific sections with a summary section of what the reader would find in the specific provider process